src/cgame/default/viewcontrollers/KeysViewController.c

```diff
@@ -104,8 +104,8 @@ static void loadView(ViewController *self) {
 
 			Cg_BindInput((View *) stackView, "Attack", "+attack");
 			Cg_BindInput((View *) stackView, "Grapple Hook", "+hook");
-			Cg_BindInput((View *) stackView, "Next weapon", "weapon_next");
-			Cg_BindInput((View *) stackView, "Previous weapon", "weapon_previous");
+			Cg_BindInput((View *) stackView, "Next weapon", "cg_weapon_next");
+			Cg_BindInput((View *) stackView, "Previous weapon", "cg_weapon_previous");
```

Comment:
So you no longer need to cycle through on the server.. this means the delay is enforced client-side, right? An easy cheat would then be to remove the delay client-side, yea?

Not a show-stopper for me, just want to fully understand.